lib/driver.js

```diff
@@ -521,6 +521,7 @@ export class XCUITestDriver extends BaseDriver {
         `Normalized platformVersion capability value '${this.opts.platformVersion}' to '${normalizedVersion}'`,
       );
       this.opts.platformVersion = normalizedVersion;
+      this.caps.platformVersion = this.opts.platformVersion;
```

Comment:
please move this expression out of the `if` block. We want it to be always executed

Reply:
Done. Thanks, I slipped there while copy-pasting this. 🙈